src/main/java/com/fasterxml/uuid/impl/LoggerFacade.java

```diff
@@ -1,10 +1,11 @@
 package com.fasterxml.uuid.impl;
 
+import com.fasterxml.uuid.Jug;
```

Comment:
Why?

Reply:
My bad, this is an incorrect IDEA IDE auto-import that I missed

Reply:
Ah. Sorry, should have guessed it's just IDE auto-import.